style.css

```diff
@@ -0,0 +1,127 @@
+html {
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+body {
+  font-family: sans-serif;
+  line-height: 1.42857143;
+  color: rgb(238, 229, 229);
+  margin: 0;
+  padding: 0;
+}
+
+.flex-nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid black;
+  background-color: rgb(65, 60, 54);
+}
+
+.main-head {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5px 10px;
+  padding: 5px 10px;
+  background-color: rgb(65, 60, 54);
+}
+
+.main-head > h1 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 24px;
+  text-align: center;
+}
+
+.menu-icon {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5px 10px;
+  padding: 5px 10px;
+  background-color: rgb(65, 60, 54);
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background-color: rgba(28, 27, 27, 0.945);
+}
```

Comment:
- [ ] The spacing is more in your design so kindly make less spacing to match the design.
- [ ] The icons are on the right so Kindly make is on the left to match the Figma design
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/170523920-9353807b-f9c0-4147-a042-bc7184049b63.png)
### your design
![hiisii](https://user-images.githubusercontent.com/81727988/170523964-1d18af58-cc2b-4d71-81f2-1ff796649a47.png)